source/configure/authentication-configuration-settings.rst

```diff
@@ -2540,21 +2540,19 @@ Access the following configuration settings in the System Console by going to **
   :configjson: Enable
   :environment: N/A
 
-  - **true**: Allow guest invitations to channels within teams.
-  - **false**: **(Default)** Email signup is disabled. This limits signup to Single sign-on services like OAuth or AD/LDAP.
+  - **true**: Enables the guest account feature.
+  - **false**: **(Default)** Disables the guest account feature.
 
 Enable guest access
 ~~~~~~~~~~~~~~~~~~~
 
 *Available in legacy Enterprise Edition E10 and E20*
 
-**True**: **(Default for Cloud deployments)** Allow guest invitations to channels within teams. Please see `Guest Accounts documentation <https://docs.mattermost.com/onboard/guest-accounts.html>`__ for more information.
-
-**False**: **(Default for self-hosted deployments)** Email signup is disabled. This limits signup to Single sign-on services like OAuth or AD/LDAP.
-
-+----------------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"Enable": false`` with options ``true`` and ``false``. |
-+----------------------------------------------------------------------------------------------------+
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------+
+| - **true**: Enables the guest account feature. See `Guest Accounts <https://docs.mattermost.com/onboard/guest-accounts.html>`__.                                                | - System Config path: **Authentication > Guest Access**             |
+| - **false**: **(Default)** Disables the guest account feature. See `Disable Guest Accounts <https://docs.mattermost.com/onboard/guest-accounts.html#disable-guest-accounts>`__. | - ``config.json`` setting: ``.GuestAccountsSettings.Enable: false`` |
```

Comment:
Please see https://github.com/mattermost/docs/pull/6172#discussion_r1085382223 for guidance on the format for page link references in the docs.